requirements/frozen.txt

```diff
@@ -132,6 +132,7 @@ Pygments==2.2.0
 PyJWT==1.5.0
 pyparsing==2.2.0
 pytest==3.2.0
+pytest-repeat==0.4.1
```

Comment:
**required** a unpinned dependency in the requirement template

Reply:
@RonnyPfannschmidt could you pls elaborate?

Reply:
the frozen files are regenerated from the requirement templates on occasion freezing new versions of everything, leaving new packages out of that means we open up for a regression

Reply:
I see! Thank you for an explanation, I will edit the accordingly